go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/fahedouch/go-logrotate v0.2.0
 	github.com/fatih/color v1.16.0
 	github.com/fluent/fluent-logger-golang v1.9.0
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
You can use errors.Join
https://github.com/search?q=repo%3Acontainerd%2Fnerdctl%20errors.Join&type=code